integration/delete_test.go

```diff
@@ -55,3 +56,145 @@ func TestDeleteSimple(t *testing.T) {
 		})
 	}
 }
+
+// TestDeleteOrdered checks if ordered parameter works properly.
+func TestDeleteOrdered(t *testing.T) {
+	setup.SkipForTigris(t)
```

Comment:
Do we still need that test? If yes, why we skip it for Tigris?

Reply:
We skip it because tigris has an issues with some types that are in shareddata, however I think that this test might be unnecessary and we can remove it.